### ISSUE TITLE
Idea: Allow support for 'maintenance' branches (git branch from x commit)

### ISSUE THREAD

**Issue by Sharor** [Praqma/git-phlow#104]
Universal Robots brought this one up, and even coded it with their 'own' git phlow setup - 

Basically you need to be able to go back to a commit, and setup up a new 'master' in the form of a maintenance branch. 

Ie commit HEAD~5 becomes a maintenance branch, I work on it a bit and deliver a ready branch which targets this new branch.

**Comment by groenborg** [comment]
This feature would require that phlow supported multiple integration branches if I understand this correctly? @Sharor

**Comment by Sharor** [comment]
Yeah basically it requires phlow to support setting a different branch as master, than master :wink:

**Comment by groenborg** [comment]
@Sharor Good idea! This issue is related to #45 we need to support more configurations so git phlow is not locked on one integration branch

**Comment by groenborg** [comment]
@Sharor the next time we meet I would like a demo of this setup that you implemented at UR
